src/test/java/com/google/api/generator/gapic/composer/GrpcServiceCallableFactoryClassComposerTest.java

```diff
@@ -35,6 +38,10 @@
 public class GrpcServiceCallableFactoryClassComposerTest {
   private ServiceDescriptor echoService;
   private FileDescriptor echoFileDescriptor;
+  private static final String GOLDENFILES_DIRECTORY =
```

Comment:
Consider having a `ComposerConstants` class in this package, and putting this constant there instead.

Reply:
Done, also moved `TESTFILES_DIRECTORY` there for share use.